### ISSUE TITLE
Validation against strict or loose URIs to be made configurable

### ISSUE THREAD

**Issue by angiolep** [angiolep/akka-wamp#7]
Add the following TypeSafe configuration item for our ``akka-wamp`` library

```bash
akka {
  wamp {
    serialization {
      # The boolean switch (default is false) to validate against strict URIs 
      # rather than loose URIs
      #
      validate-strict-uris = false
    }}}
```

So that, it can be configured to check against strict or loose URI as described in the specification [section 5.1.1](https://raw.githubusercontent.com/wamp-proto/wamp-proto/master/rfc/draft-oberstet-hybi-tavendo-wamp.txt)

**Comment by angiolep** [comment]
There's a discussion going on about this

https://github.com/wamp-proto/wamp-proto/issues/115

**Comment by angiolep** [comment]
Fixed in v0.6.0
